crates/config/src/lib.rs

```diff
@@ -429,8 +434,10 @@ pub struct Config {
     #[serde(default)]
     pub extra_output_files: Vec<ContractOutputSelection>,
     /// Whether to print the names of the compiled contracts.
+    // todo(onbjerg): this seems unused.
     pub names: bool,
     /// Whether to print the sizes of the compiled contracts.
+    // todo(onbjerg): this seems unused.
     pub sizes: bool,
```

Comment:
These should not be unused; should correspond with the `forge build --sizes` / `forge build --names`  / `forge build --names --sizes` flags

I am not sure if this ever worked

https://github.com/morpho-org/morpho-optimizers/blob/04143d929372e53d3f6822f62ee90df1609fc784/foundry.toml#L9-L10